howto/finetune_adapter.md

```diff
@@ -27,6 +27,7 @@ or [prepare your own dataset](#tune-on-your-dataset).
 python finetune_adapter.py --checkpoint_dir checkpoints/stabilityai/stablelm-base-alpha-3b
 ```
 
+The finetuning requires at least one GPU with ~24 GB memory (GTX 3090).
```

Comment:
Is this number updated for the 3b stablelm model?

Reply:
with micro_batch_size = 4

Reply:
Thank you!